Cargo.toml

```diff
@@ -143,7 +143,7 @@ fedimint-wallet-common = { path = "./modules/fedimint-wallet-common" }
 #             https://github.com/ipetkov/crane/issues/370
 [profile.dev.build-override]
 debug = false
-opt-level = 1
+opt-level = 3
```

Comment:
Should we also change `profile.ci.build-override.opt-level`?

Reply:
I think ci inherits everything from dev, but ... I wonder how to really test it.

Reply:
I believe we'll need to include `inherits = "dev"` in the ci profile.

https://doc.rust-lang.org/cargo/reference/profiles.html#custom-profiles

Reply:
@bradleystachurski We do already.

Reply:
:facepalm: we do! Carry on.